spec/researches/getSentenceBeginningsSpec.js

```diff
@@ -2,66 +2,125 @@ var sentenceBeginnings = require( "../../js/researches/getSentenceBeginnings.js"
 		expect( sentenceBeginnings( mockPaper )[2].word ).toBe( "hey" );
 		expect( sentenceBeginnings( mockPaper )[2].count ).toBe( 1 );
 	} );
-	it( "returns an object with sentence beginnings and counts for three sentences all starting with one of the exception words.", function() {
-		var mockPaper = new Paper( "The boy, hey! The boy. The boy." );
+	it( "returns an object with sentence beginnings and counts for three sentences in English all starting with one of the exception words.", function() {
+		var mockPaper = new Paper( "The boy, hey! The boy. The boy.", { locale: 'en_US'} );
 		expect( sentenceBeginnings( mockPaper )[0].word ).toBe( "the boy" );
 		expect( sentenceBeginnings( mockPaper )[0].count ).toBe( 3 );
 	} );
-	it( "returns an object with sentence beginnings and counts for three sentences all starting with one of the exception words. The second word of all sentences is also in the list " +
+	it( "returns an object with sentence beginnings and counts for three sentences in English all starting with one of the exception words. The second word of all sentences is also in the list " +
 		"of exception words, which should not matter.", function() {
-		var mockPaper = new Paper( "One, two, three. One, two, three. One, two, three." );
+		var mockPaper = new Paper( "One, two, three. One, two, three. One, two, three.", { locale: 'en_US'} );
 		expect( sentenceBeginnings( mockPaper )[0].word ).toBe( "one two" );
 		expect( sentenceBeginnings( mockPaper )[0].count ).toBe( 3 );
 	} );
-
-	it( "returns an object with sentence beginnings in lists", function() {
+	it( "returns an object with sentence beginnings and counts based on the default (English) when no locale is included.", function() {
+		var mockPaper = new Paper( "The boy, hey! The boy. The boy." );
+		expect( sentenceBeginnings( mockPaper )[0].word ).toBe( "the boy" );
+		expect( sentenceBeginnings( mockPaper )[0].count ).toBe( 3 );
+	} );
+	it( "returns an object with sentence beginnings and counts based on the default (English) when a non-existing locale is included.", function() {
+		var mockPaper = new Paper( "The boy, hey! The boy. The boy.", { locale: 'xx_yy'} );
+		expect( sentenceBeginnings( mockPaper )[0].word ).toBe( "the boy" );
+		expect( sentenceBeginnings( mockPaper )[0].count ).toBe( 3 );
+	} );
+	it( "returns an object with sentence beginnings and counts for two sentences in French starting with different words.", function() {
+		var mockPaper = new Paper( "Sur le pont d'Avignon. Liberté, égalité, fraternité. ", { locale: 'fr_FR'} );
+		expect( sentenceBeginnings( mockPaper )[0].word ).toBe( "sur" );
+		expect( sentenceBeginnings( mockPaper )[0].count ).toBe( 1 );
+		expect( sentenceBeginnings( mockPaper )[1].word ).toBe( "liberté" );
+		expect( sentenceBeginnings( mockPaper )[1].count ).toBe( 1 );
+	} );
+	it( "returns an object with sentence beginnings and counts for two sentences in French starting with the same word.", function() {
```

Comment:
I would love some whitespace between the end of the previous spec and the start of the next.